girder/girder_large_image/web_client/vue/components/FrameSelector.vue

```diff
@@ -189,6 +189,7 @@ export default Vue.extend({
                 :currentValue="indexInfo[index].current"
                 :valueMax="indexInfo[index].range"
                 :label="index.replace('Index', '')"
+                :sliderLabels="index === 'IndexC' ? imageMetadata.channels : []"
```

Comment:
I think we need to add `:sliderLabels="[]"` to the mode 0 block above or adjust the DualInput component to check if it exists.